crates/rethnet_eth/src/remote/client.rs

```diff
@@ -749,6 +749,26 @@ mod tests {
             }
         }
 
+        #[tokio::test]
+        async fn get_earliest_block() {
+            let alchemy_url = get_alchemy_url();
+
+            let _block = RpcClient::new(&alchemy_url)
+                .get_block_by_number(BlockSpec::earliest())
+                .await
+                .expect("should have succeeded");
+        }
+
+        #[tokio::test]
+        async fn get_earliest_block_with_transaction_data() {
+            let alchemy_url = get_alchemy_url();
+
+            let _block = RpcClient::new(&alchemy_url)
+                .get_block_by_number_with_transaction_data(BlockSpec::earliest())
+                .await
+                .expect("should have succeeded");
+        }
+
```

Comment:
For these, why not validate something in the return data? Since it will always be the same. Perhaps block number or block hash?